src/cr/cube/cube.py

```diff
@@ -645,6 +705,15 @@ class _UnweightedCountMeasure(_BaseMeasure):
     @lazyproperty
     def _flat_values(self):
         """tuple of int counts before weighting."""
+        # ---If valid_count are expressed in the cube dict, returns its data---
+        if (
+            self._cube_dict["result"]["measures"]
```

Comment:
I'm not sure, but is it possible _not_ to have measures? I honestly don't know :)

Reply:
i don't think it's possible